whitesource/client.py

```diff
@@ -64,41 +64,47 @@ def save_project_tag(self, projectToken: str, key: str, value: str):
             json=body,
         )
 
-    def post_project(
+    async def ws_project(
         self,
         extra_whitesource_config: typing.Dict,
         file,
-        filename: str,
         project_name: str,
         requester_email: str,
+        length: int,
+        chunk_size: int,
```

Comment:
I think this should be the _only_ place (for now) where where hard-code the chunk_size, ping_interval, and ping_timeout.